recipes/hiredis/all/conandata.yml

```diff
@@ -2,7 +2,13 @@ sources:
   "1.0.0":
     url: "https://github.com/redis/hiredis/archive/v1.0.0.tar.gz"
     sha256: "2a0b5fe5119ec973a0c1966bfc4bd7ed39dbce1cb6d749064af9121fe971936f"
+  "1.1.0-rc":
+    url: "https://github.com/ayeganov/hiredis/archive/v1.1.0-rc.tar.gz"
```

Comment:
unfortunately we do not accept forks making unofficial releases.

Reply:
That makes sense. I didn't read the FAQ as carefully, so didn't notice the unofficial package notation.